pkg/impl/libraptorq/encoder.go

```diff
@@ -12,7 +12,7 @@ type EncoderFactory struct {
 
 func (*EncoderFactory) New(input []byte, symbolSize uint16, minSubSymbolSize uint16,
 	maxSubBlockSize uint32, alignment uint8) (enc raptorq.Encoder, err error) {
-	wrapped := swig.NewBytesEncoder(input, minSubSymbolSize, symbolSize,
+	wrapped := swig.InitBytesEncoder(input, minSubSymbolSize, symbolSize,
```

Comment:
any comment here?

Reply:
InitBytesEncoder = NewBytesEncoder (BytesEncoder ctor) call + its `compute(…)` method call.  See the inlined definition in `swig.swigcxx` for details.